pkgs/applications/misc/prusa-slicer/super-slicer.nix

```diff
@@ -0,0 +1,48 @@
+{
+  stdenv, lib, fetchFromGitHub, makeDesktopItem, prusa-slicer
+}:
+let
+  appname = "SuperSlicer";
+  version = "2.2.53.1";
+  pname = "super-slicer";
+  override = super: {
+    inherit version pname;
+
+    src = fetchFromGitHub {
+      owner = "supermerill";
+      repo = "SuperSlicer";
+      hash = "sha256-CAhwmQ63N/XJYToTnIV84lNnjDGNbkmYPzNKNL/wVxs=";
+      rev = "${version}";
+    };
+
+    # See https://github.com/supermerill/SuperSlicer/issues/432
+    cmakeFlags = super.cmakeFlags ++ [
+      "-DSLIC3R_BUILD_TESTS=0"
+    ];
+
+    postInstall = ''
+    mkdir -p "$out/share/pixmaps/"
+    ln -s "$out/share/slic3r++/icons/Slic3r.png" "$out/share/pixmaps/${appname}.png"
+    mkdir -p "$out/share/applications"
+    cp "$desktopItem"/share/applications/* "$out/share/applications/"
+    '';
+
+    desktopItem = makeDesktopItem {
+      name = appname;
+      exec = "superslicer";
+      icon = appname;
+      comment = "G-code generator for 3D printers";
+      desktopName = appname;
+      genericName = "3D printer tool";
+      categories = "Development;";
+    };
+
+    meta = with stdenv.lib; {
+      description = "G-code generator for 3D printer";
+      homepage = "https://github.com/supermerili/SuperSlicer";
+      license = licenses.agpl3;
+      maintainers = with maintainers; [ cab404 ];
```

Comment:
You can add me (moredread) if you like :)

Reply:
Would be glad to)

Reply:
Added!